server/routes/trace_analytics_dsl_router.ts

```diff
@@ -137,7 +151,7 @@ export function registerTraceAnalyticsDslRouter(router: IRouter, dataSourceEnabl
       const { index, size, ...rest } = request.body;
```

Comment:
Should be modified to `const { index, size, tenant, ...rest } = request.body;`

Reply:
good catch. it should extract tenant, i'll update PR